src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -106,5 +108,26 @@ public static void DeleteHitsFile(string path)
 
             RetryHelper.Retry(() => File.Delete(path), retryStrategy, 10);
         }
+        
+        public static string[] GetExcludedFiles(string[] exclusionRules, string parentDir) {
+            if (exclusionRules == null || exclusionRules.Length == 0 ) return null;
+            var matcher = new Matcher();
+            foreach (var exclusionRule in exclusionRules)
+            {
+                matcher.AddInclude(exclusionRule);
+            }
+        
+            DirectoryInfo directoryInfo = new DirectoryInfo(parentDir);
+            
+            var fileMatchResult = matcher.Execute(new DirectoryInfoWrapper(directoryInfo));
+            return fileMatchResult.Files
+                .Select(
+                    f => System.IO.Path.GetFullPath(
+                        System.IO.Path.Combine(directoryInfo.ToString(), f.Path)
```

Comment:
i think  instead of `directoryInfo.ToString()`  you can use `parentDir` right?
https://source.dot.net/#System.IO.FileSystem/System/IO/DirectoryInfo.cs,29
https://source.dot.net/#System.IO.FileSystem/System/IO/FileSystemInfo.cs,108

Reply:
nit: you can avoid `System.IO.` and use `Path.xxx`(using `System.IO;` )